shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -74,6 +75,28 @@
 
 #pragma mark - Static Functions
 
+static CFMutableCharacterSetRef EmojiCharacterSet() {
+  static CFMutableCharacterSetRef set = nil;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    set = CFCharacterSetCreateMutableCopy(
+        kCFAllocatorDefault,
+        CTFontCopyCharacterSet(CTFontCreateWithName(CFSTR("AppleColorEmoji"), 0.0, NULL)));
```

Comment:
If we found out other characters than "AppleColorEmoji" also causes this issue, we can add them to this set.